cfgov/cfgov/test.py

```diff
@@ -28,3 +29,15 @@ def setup_wagtail_root(self):
 
         module = importlib.import_module(migration)
         module.initial_data(apps, None)
+
+
+class HtmlMixin(object):
```

Comment:
Created this useful mixin for use in unit tests where you are comparing generated HTML.